From da9305cf2e9297411cd74a35d5644fd61d039f2d Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Sat, 14 Oct 2023 14:59:37 -0500
Subject: [PATCH 07/14] journalctl: Only show info messages and above

Some users are bothered by the verbose debug output that some daemons have. Rather than removing what is potentially useful information from
the journal instead change the default priority of journalctl so they don't see it by default.
---
 man/journalctl.xml       | 3 ++-
 src/journal/journalctl.c | 2 +-
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/man/journalctl.xml b/man/journalctl.xml
index 70d8508cf0..278197fd9c 100644
--- a/man/journalctl.xml
+++ b/man/journalctl.xml
@@ -344,7 +344,8 @@
         messages with this log level or a lower (hence more important) log level are shown. If a range is
         specified, all messages within the range are shown, including both the start and the end value of the
         range. This will add <literal>PRIORITY=</literal> matches for the specified
-        priorities.</para></listitem>
+        priorities. SOLUS NOTE: If this is not specified then only messages of info and above will be shown by
+        by default. You need to use <literal>--priority=7</literal> if you want to see debug messages.</para></listitem>
       </varlistentry>
 
       <varlistentry>
diff --git a/src/journal/journalctl.c b/src/journal/journalctl.c
index f2f3918382..cc695c935d 100644
--- a/src/journal/journalctl.c
+++ b/src/journal/journalctl.c
@@ -111,7 +111,7 @@ static bool arg_show_cursor = false;
 static const char *arg_directory = NULL;
 static char **arg_file = NULL;
 static bool arg_file_stdin = false;
-static int arg_priorities = 0xFF;
+static int arg_priorities = 0x7F; // Info and above
 static Set *arg_facilities = NULL;
 static char *arg_verify_key = NULL;
 #if HAVE_GCRYPT
-- 
2.43.0

